udaykalra/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -40,11 +61,20 @@
     availableRanges.add(fullDay);
 
     // Obtain data from the request.
-    Collection<String> attendees = request.getAttendees();
+    Collection<String> attendees = new HashSet<>();
+    Collection<String> mandatoryAttendees = request.getAttendees();
+    Collection<String> optionalAttendees = request.getOptionalAttendees();
```

Comment:
It looks like this could be declared within the `if (withOptional)` block. Limiting the scope of variables is preferred, to avoid misuse.

Reply:
Done